tests/testthat/test_tex_catwitherror.R

```diff
@@ -30,7 +30,7 @@ test_that('large_error', {
 
 test_that('similar_error', {
     expect_equal(tex.catwitherror(7492.8291130334482659, 1759.0859320695926726,
-                                  digits = 3, with.dollar = FALSE), '7492.829(090)')
+                                  digits = 3, with.dollar = FALSE), '7492(1759)')
```

Comment:
oops, I had very little sleep the last few days! cheers

Reply:
So, `tex.catwitherror` is now doing the right thing (to three digits) and we should adjust the test:

```
  tex.catwitherror(...) not equal to "7492(1759)".
  1/1 mismatches
  x[1]: "7493(1759)"
  y[1]: "7492(1759)"
```

Reply:
done

Reply:
This happens when you try to do a bugfix in 5 mins.